client/tests/__mocks__/GraphQLMocks/TestPlanReportStatusDialogMock.js

```diff
@@ -301,7 +301,82 @@ export default (meQuery, testPlanReportStatusDialogQuery) => [
+                ats: [
+                    {
+                        id: '1',
+                        name: 'JAWS',
+                        atVersions: [
+                            {
+                                id: '1',
+                                name: '2021.2111.13',
+                                releasedAt: '2021-11-01T04:00:00.000Z'
+                            }
+                        ],
+                        browsers: [
+                            {
+                                id: '2',
+                                name: 'Chrome'
+                            },
+                            {
+                                id: '1',
+                                name: 'Firefox'
+                            }
+                        ],
+                        candidateBrowsers: [{ id: '2' }],
+                        recommendedBrowsers: [{ id: '1' }, { id: '2' }]
+                    },
+                    {
+                        id: '2',
+                        name: 'NVDA',
+                        atVersions: [
+                            {
+                                id: '2',
+                                name: '2020.4',
+                                releasedAt: '2021-02-19T05:00:00.000Z'
+                            }
+                        ],
+                        browsers: [
+                            {
+                                id: '2',
+                                name: 'Chrome'
+                            },
+                            {
+                                id: '1',
+                                name: 'Firefox'
+                            }
+                        ],
+                        candidateBrowsers: [{ id: '2' }],
+                        recommendedBrowsers: [{ id: '1' }, { id: '2' }]
+                    },
+                    {
+                        id: '3',
+                        name: 'VoiceOver for macOS',
+                        atVersions: [
+                            {
+                                id: '3',
+                                name: '11.6 (20G165)',
+                                releasedAt: '2019-09-01T04:00:00.000Z'
+                            }
+                        ],
+                        browsers: [
+                            {
+                                id: '2',
+                                name: 'Chrome'
+                            },
+                            {
+                                id: '1',
+                                name: 'Firefox'
+                            },
+                            {
+                                id: '3',
+                                name: 'Safari'
+                            }
+                        ],
+                        candidateBrowsers: [{ id: '3' }],
+                        recommendedBrowsers: [{ id: '2' }, { id: '3' }]
+                    }
+                ]
```

Comment:
Having looked at this together yesterday, I was wondering if you would be able to remove these changes and check if the tests still pass? I don't recall whether this is needed or not.

Reply:
I think they are needed, but the tests pass without them.

Reply:
Since these are only used in tests, I think if the tests pass that's pretty much confirms my suspicion that they're not needed. Would you mind removing those changes?